src/views/CompareWithAzion/CompareView.vue

```diff
@@ -226,13 +239,23 @@
         secureData.value = parseSecureData(medianFirstView)
       })
       .catch(() => {
-        errorData.value = 'Not possible to load the Origin test.'
+        setTestMessage({
+          message: 'Not possible to load the Origin test.',
+          severity: 'error',
+          retry: false
+        })
       })
+  }
 
-    getResultFromWebpagetest(testById.body[0].id_azion)
+  const loadTestWithAzion = async (id) => {
+    return getResultFromWebpagetest(id)
       .then((resutWithAzion) => {
         if (resutWithAzion.body.statusCode !== 200) {
-          errorDataWithAzion.value = `Test ${resutWithAzion.body.data.id} is ${resutWithAzion.body.statusText}`
+          setTestWithAzionMessage({
+            message: `Test ${resutWithAzion.body.data.id} is ${resutWithAzion.body.statusText}`,
+            severity: 'info',
+            retry: true
+          })
           return
         }
```

Comment:
O correto séria o serviço retornar o texto da mensagem formatada e não ter todo o trabalho de formatação na view

Reply:
aqui o status é 200, a resposta veio correta só que a API disse que está sendo executado ainda, se for no service precisaria passar o info da para saber o tipo de servery

Reply:
O padrão que seguimos é implementar o adapter do retorno da API no serviço. Dessa forma, o retorno do serviço será a message, permitindo que você avalie se a mensagem foi preenchida através de uma condicional.

É importante ressaltar que este é apenas um exemplo e que poderia haver outras abordagens. No entanto, o essencial é remover a responsabilidade da view em realizar o adapter do retorno do serviço.